files/install-ptfe.sh

```diff
@@ -185,6 +197,20 @@ if [[ $(< /etc/ptfe/airgap-installer-url) != none ]]; then
     airgap_installer_url_path="/etc/ptfe/airgap-installer-url"
 fi
 
+if [[ $(< /etc/ptfe/custom-ca-cert-url) != none ]]; then
+  custom_ca_cert_url=$(cat /etc/ptfe/custom-ca-cert-url)
+  custom_ca_cert_file_name=$(echo "${custom_ca_cert_url}" | awk -F '/' '{ print $NF }')
+  # Setting up a tmp directory to do this `jq` transform to leave artifcats if anything goes "boom".
+  mkdir -p /tmp/ptfe/customer-certs
+  pushd /tmp/ptfe/customer-certs
+  wget --trust-server-files "${custom_ca_cert_url}"
```

Comment:
Wondering if we check that the return code here is good and we got something? Perhaps bail with an error message if the wget failed for any reason?